Editor/AGS.Editor/Panes/InventoryEditor.cs

```diff
@@ -48,22 +77,40 @@ public void RefreshData()
         private void UpdateControlsEnabled()
         {
             currentItemGroupBox.Visible = (_item != null);
+            UpdatePanelSizes();
         }
 
         private void pnlCursorImage_Paint(object sender, PaintEventArgs e)
         {
             if (_item != null)
             {
-                IMAGE_SCALE_FACTOR = Factory.AGSEditor.CurrentGame.GUIScaleFactor;
-
                 IntPtr hdc = e.Graphics.GetHdc();
+                if (_previousItemCursorImage != _item.CursorImage) UpdatePanelSizes();
                 Size spriteSize = Utilities.GetSizeSpriteWillBeRenderedInGame(_item.CursorImage);
-                Factory.NativeProxy.DrawSprite(hdc, 0, 0, spriteSize.Width * IMAGE_SCALE_FACTOR, spriteSize.Height * IMAGE_SCALE_FACTOR, _item.CursorImage);
+                Factory.NativeProxy.DrawSprite(hdc, 0, 0,
+                    spriteSize.Width * zoomSlider.Value, spriteSize.Height * zoomSlider.Value,
+                    _item.CursorImage);
                 e.Graphics.ReleaseHdc();
-                if ((_item.HotspotX > 0) && (_item.HotspotY > 0))
+                if ((_item.HotspotX > -1) && (_item.HotspotY > -1))
                 {
-                    e.Graphics.DrawLine(Pens.LightGreen, (_item.HotspotX - 2) * IMAGE_SCALE_FACTOR, _item.HotspotY * IMAGE_SCALE_FACTOR, (_item.HotspotX + 2) * IMAGE_SCALE_FACTOR, _item.HotspotY * IMAGE_SCALE_FACTOR);
-                    e.Graphics.DrawLine(Pens.Blue, _item.HotspotX * IMAGE_SCALE_FACTOR, (_item.HotspotY - 2) * IMAGE_SCALE_FACTOR, _item.HotspotX * IMAGE_SCALE_FACTOR, (_item.HotspotY + 2) * IMAGE_SCALE_FACTOR);
+                    Pen penGreen = new Pen(Color.LightGreen, 1);
+                    Pen penBlue = new Pen(Color.Blue, 1);
+
+                    // Create rectangle.
+                    Rectangle rectH = new Rectangle(

```

Comment:
If you feel the parameters are not obvious what they are you can specify this with a syntax feature instead of comments

```suggestion
                    Rectangle rectH = new Rectangle(
                        x: (_item.HotspotX - 1) * zoomSlider.Value,
                        y: (_item.HotspotY ) * zoomSlider.Value,
                        width: 3 * zoomSlider.Value,
                        height: zoomSlider.Value);
```